### DISCUSSION TITLE
FOSDEM 2024

### DISCUSSION THREAD

**Discussion by Ralim** [Ralim/IronOS#1865] [General]
This is just as a note that I (Ralim/Ben) will be at Fosdem 2024.

Pine64 has covered the costs for me to be ble to attend the event, so I will be likely around the Pine64 booth + at any Pine events.

I am there for _IronOS_ however, so of course feel free to find me for anything IronOS related, even if its not a Pine64 product ^_^.

If you will be around the event feel free to contact me by email to get a better means of contact etc. (use same email as my git details -- ralim<at>ralimtek.com).

A big thankyou to Pine though as travel from Australia isnt _cheap_.

**Comment by ia** [comment]
Sincerely wish you a good trip: have interesting & entertaining time there!

> I will be likely around the Pine64 booth + at any Pine events
> I am there for IronOS however, so of course feel free to find me for anything IronOS related

Just out of curiosity - are you going to give any [technical] talk there (at least short one, with presentation, recording & stuff)? Or is it more like _community meeting_-like events?
